src/ui/field_formats_mixin.js

```diff
@@ -0,0 +1,7 @@
+import { FieldFormatsService } from '../core_plugins/kibana/common/field_formats/field_formats';
+import { getUiSettingDefaults } from '../core_plugins/kibana/ui_setting_defaults';
+
+export function fieldFormatsMixin(kbnServer, server) {
+  server.decorate('server', 'uiSettingDefaults', getUiSettingDefaults());
```

Comment:
Never mind. Spencer straightened my out. After a rebase, getDefaults is populated and spitting out the correct values. I will remove the uiSettingsDefaults decorator.